_templates/build.yml.erb

```diff
@@ -39,6 +39,6 @@ jobs:
         password: ${{ secrets.DOCKER_PASSWORD }}
         add_git_labels: true
         target: ${{ matrix.target }}
-        repository: snyk/snyk
+        repository: ${{ secrets.DOCKER_USERNAME }}/snyk
```

Comment:
Maintainers my want you to remove this change, since its only needed for building in your fork, not master

Reply:
Yeah, I had put that in there so anyone with a fork would be able to work w/out modification.  We'll see what the Snyk engineering folks think. (I'm at Snyk but am "just" a dev advocate, so not in the trenches :) 
Thanks for the input.

Reply:
It's ok :)